src/cmd/validators/unstake.rs

```diff
@@ -42,6 +53,7 @@ impl Cmd {
                     .await?
                     .stake
             },
+            stake_release_height: self.stake_release_height,
```

Comment:
should we recommend / provide a fall back?

Reply:
no because the intent is to enforce the user to declare this.. We could try to calculate one by talking to the api, but then it's optional again and wouldn't work in air-gap setups